### DISCUSSION TITLE
Validation rules not working with exclude_unless / exclude_if Validation Rule for Resource

### DISCUSSION THREAD

**Discussion by airdev-web** [laravel/nova-issues#3501] [Q&A]
Greetings from **Belgium** !

- Laravel Version: 8.51.0
- Nova Version: 3.26.1
- PHP Version: 7.4

### Description:
Can't use Laravel Validation rules `exclude_unless` (or even with `exclude_if`).
I need my field `allergens` to be required only if my field `is_active` is `true`.

Following code doesn't seem to work. No error code, but the validation doesn't work.

### Detailed steps to reproduce the issue on a fresh Nova installation:
```php
Boolean::make('Actif', 'is_active'),
Textarea::make('Allergènes', 'allergens')
    ->rules('exclude_if:is_active,true|required'),

// I also tried this
Textarea::make('Allergènes', 'allergens')
    ->rules('exclude_if:is_active,true', 'required'),
```

Thanks for the ideas or correct usage,
Romain

**Comment by crynobone** [comment]
Boolean submit `1` and `0` to the backend, I don't think `true` would work here.
